tests/dummy/app/pods/link/template.hbs

```diff
@@ -1,292 +1,296 @@
 <div class="dummy-body link">
-    <div class="section-title">Primary</div>
-    <div class="section">
-        <div class="example">
-            <div class="title">Disabled</div>
-            <div class="demo">
-              {{#frost-link
-              'link.min'
-              priority='primary'
-              size='small'
-              disabled=true
-              }}
-                  <div class='text'>Primary</div>
-              {{/frost-link}}
-            </div>
-            <div class="snippet">
-              {{format-markdown "```handlebars
-{{#frost-link
-  'link.min'
+  <div class="section-title">Primary</div>
+
+  <div class="section">
+
+    <div class="example">
+      <div class="title">Disabled</div>
+      <div class="demo">
+        {{#frost-link 'link.min'
+          priority='primary'
+          size='small'
+          disabled=true
```

Comment:
Can we flip disabled and small/medium/large?  I'd like to show small/medium/large first